### DISCUSSION TITLE
Community Driver Listings on Website Outdated - Please Consider kyagie/laravel-auditing-filesystem

### DISCUSSION THREAD

**Discussion by kyagie** [owen-it/laravel-auditing#971] [Ideas]
Hi,
I noticed that the list of community drivers for the owen-it/laravel-auditing package on the official website contains several outdated drivers. Specifically, some of them reference repositories that are no longer actively maintained or compatible with recent versions of Laravel.

I’ve created a new filesystem driver package, **kyagie/laravel-auditing-filesystem**, which is up-to-date and compatible with the latest versions of PHP and Laravel. This package allows the storage of audit logs in CSV format across all registered Laravel disks.

It is based on a fork of the original [betapeak/laravel-auditing-filesystem](https://github.com/betapeak/laravel-auditing-filesystem), but with the following improvements:

Support for higher PHP versions (PHP 8.1 and above).
Updated dependencies to align with the latest Laravel and Auditing packages.
You can find the package here: [kyagie/laravel-auditing-filesystem](https://github.com/kyagie/laravel-auditing-filesystem).

I’d appreciate it if you could consider adding my driver to the community drivers list and reviewing the current outdated listings.

Thanks for your time, and please let me know if any further information is needed.

Best regards,
Kyagulanyi Arthur

**Comment by willpower232** [comment]
If you make a PR over at https://github.com/owen-it/laravel-auditing.com then someone can double check and update the site at the same time.

**Reply by kyagie** [reply]
Thanks
